packages/storybook-addon-web-components-knobs/demo/stories/demo-wc-card.stories.mdx

```diff
@@ -1,4 +1,4 @@
-import { Story, Preview, Meta, Props } from '@storybook/addon-docs/blocks';
+import { Story, Preview, Meta, Props } from '@open-wc/demoing-storybook';
```

Comment:
I am afraid here the original is correct as we are not using `@open-wc/demoing-storybook` in that packages...